src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  flex-shrink: 0;
+  flex-wrap: wrap;
+  gap: $gap-between-columns;
+  padding: $calendar-padding;
+  width: ($column-size * $columns-in-row) +
+    ($gap-between-columns * $gaps-in-row) +
+    $calendar-padding;
+
+  @for $days from 28 through 31 {
+    &--month-length--#{$days} {
+      :nth-child(n + #{$days+1}) {
+        visibility: hidden;
+      }
+    }
+  }
+
+  @for $i from 1 through 31 {
+    :nth-child(#{$i})::before {
+      content: "#{$i}";
+      display: block;
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      line-height: $column-size;
+      text-align: center;
+    }
+  }
+
+  @each $weekday, $shift in $weekdays {
+    &--start-day--#{$weekday} {
+      :first-child {
+        margin-left: #{$shift};
+      }
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
+    height: $column-size;
+    width: $column-size;
+    border: 1px solid black;
+    background-color: #eee;
+    cursor: pointer;
+    transition: transform 0.5s;
```

Comment:
What about background-color? You change it on hover too.
Also, would be great to create a variable for transition-duration as it should be the same per page
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/12399d41-b8c6-4527-a2ad-cdaced315271)